apps/wallet-mobile/src/yoroi-wallets/cardano/byron/ByronWallet.ts

```diff
@@ -697,23 +700,28 @@ export class ByronWallet implements YoroiWallet {
       throw new Error('Only one datum per transaction is supported')
     }
 
+    const {
+      coinsPerUtxoByte,
+      keyDeposit,
+      linearFee: {coefficient, constant},
+      poolDeposit,
+    } = await this.getProtocolParams()
+
     try {
       const unsignedTx = await Cardano.createUnsignedTx(
         absSlotNumber,
         addressedUtxos,
         recipients,
         changeAddr,
         {
-          keyDeposit: networkConfig.KEY_DEPOSIT,
+          keyDeposit,
           linearFee: {
-            coefficient: networkConfig.LINEAR_FEE.COEFFICIENT,
-            constant: containsDatum
-              ? String(BigInt(networkConfig.LINEAR_FEE.CONSTANT) * 2n)
-              : networkConfig.LINEAR_FEE.CONSTANT,
+            coefficient,
+            constant: containsDatum ? String(BigInt(constant) * 2n) : constant,
           },
           minimumUtxoVal: networkConfig.MINIMUM_UTXO_VAL,
-          coinsPerUtxoWord: networkConfig.COINS_PER_UTXO_WORD,
-          poolDeposit: networkConfig.POOL_DEPOSIT,
+          coinsPerUtxoWord: String(Number(coinsPerUtxoByte) * 8),
```

Comment:
is it needed the `* 8`?

Reply:
params API gives `coinsPerUtxoByte`. I assume `createUnsignedTx` will accept `coinsPerUtxoByte` in the near future right?